### DISCUSSION TITLE
nornir_netmiko connect to router with intermediate jump host

### DISCUSSION THREAD

**Discussion by phani423** [nornir-automation/nornir#843] [Q&A]
Hi ,

Can anyone share example of how to connect to router via jump host using nornir_netmiko?

Thanks!

**Comment by shekharsharma11** [comment]
Any answer you got phani ?

**Comment by ktbyers** [comment]
It is pretty much the standard Netmiko process (just implemented in a Nornir context):

https://github.com/twin-bridges/nornir_course/blob/master/bonus1/collateral/netmiko_ssh_proxy/groups.yaml#L8

https://github.com/twin-bridges/nornir_course/blob/master/bonus1/collateral/netmiko_ssh_proxy/netmiko_proxy.py

And then this process:

https://pynet.twb-tech.com/blog/netmiko-ssh-proxy-support.html
